WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/RestoreStates.kt

```diff
@@ -103,10 +104,15 @@ sealed class RestoreRequestState {
         val rewindId: String,
         val progress: Int?,
         val message: String? = null,
-        val currentEntry: String? = null
+        val currentEntry: String? = null,
+        val date: Date? = null
     ) : RestoreRequestState()
 
-    data class Complete(val rewindId: String, val restoreId: Long) : RestoreRequestState()
+    data class Complete(
+        val rewindId: String,
+        val restoreId: Long,
+        val date: Date? = null
```

Comment:
Same as above. 😄 

PS: Usually I try to avoid duplicate comment by adding a `PS: Xyz` on my first comment letting the author know that this comment applies elsewhere.

Reply:
This is done here: 4b5a9fd